pkg/commands/tls/custom/certificate/certificate_test.go

```diff
@@ -25,33 +25,63 @@ const (
 func TestTLSCustomCertCreate(t *testing.T) {
 	args := testutil.Args
 	scenarios := []testutil.TestScenario{
+		//{
+		//	Name:      "validate missing --cert-blob and --cert-path flags",
+		//	Args:      args("tls-custom certificate create"),
+		//	WantError: "neither --cert-path or --cert-blob provided, one must be provided",
+		//},
+		//{
+		//	Name:      "validate specifying both --cert-blob and --cert-path flags",
+		//	Args:      args("tls-custom certificate create --cert-blob foo --cert-file bar"),
+		//	WantError: "cert-path and cert-blob provided, only one can be specified",
+		//},
+		//{
+		//	Name:      "validate invalid --cert-file arg",
+		//	Args:      args("tls-custom certificate create --cert-file bar"),
+		//	WantError: "no such file or directory",
+		//},
+		//{
+		//	Name:      "validate nonsensical --cert-file arg",
+		//	Args:      args("tls-custom certificate create --cert-file foobar"),
+		//	WantError: "no such file or directory",
+		//},
+		// Need a test that validates CLI reads and sends a cert file correctly
 		{
-			Name:      "validate missing --cert-blob flag",
-			Args:      args("tls-custom certificate create"),
-			WantError: "required flag --cert-blob not provided",
-		},
-		{
-			Name: validateAPIError,
-			API: mock.API{
-				CreateCustomTLSCertificateFn: func(_ *fastly.CreateCustomTLSCertificateInput) (*fastly.CustomTLSCertificate, error) {
-					return nil, testutil.Err
-				},
-			},
-			Args:      args("tls-custom certificate create --cert-blob example"),
-			WantError: testutil.Err.Error(),
-		},
-		{
-			Name: validateAPISuccess,
+			Name: "validate custom cert is submitted",
 			API: mock.API{
-				CreateCustomTLSCertificateFn: func(_ *fastly.CreateCustomTLSCertificateInput) (*fastly.CustomTLSCertificate, error) {
+				CreateCustomTLSCertificateFn: func(certInput *fastly.CreateCustomTLSCertificateInput) (*fastly.CustomTLSCertificate, error) {
+					fmt.Printf("foo %s", certInput)
+					//testutil.AssertStringContains(certInput.CertBlob, "this is a fake cert")
 					return &fastly.CustomTLSCertificate{
 						ID: mockResponseID,
 					}, nil
 				},
 			},
-			Args:       args("tls-custom certificate create --cert-blob example"),
+			Args:       args("tls-custom certificate create --cert-file testdata/certificate.crt"),
 			WantOutput: fmt.Sprintf("Created TLS Certificate '%s'", mockResponseID),
 		},
+		//{
+		//	Name: validateAPIError,
+		//	API: mock.API{
+		//		CreateCustomTLSCertificateFn: func(_ *fastly.CreateCustomTLSCertificateInput) (*fastly.CustomTLSCertificate, error) {
+		//			return nil, testutil.Err
+		//		},
+		//	},
+		//	Args:      args("tls-custom certificate create --cert-blob example"),
+		//	WantError: testutil.Err.Error(),
+		//},
+		//{
+		//	Name: validateAPISuccess,
+		//	API: mock.API{
```

Comment:
You might want to try using a global variable for this (it's not pretty but it's how I've done this in the past)...

https://github.com/fastly/cli/blob/6c1ec982475e698bd347c94622f857d9e4f44c90/pkg/commands/vcl/snippet/snippet_test.go#L17

https://github.com/fastly/cli/blob/6c1ec982475e698bd347c94622f857d9e4f44c90/pkg/commands/vcl/snippet/snippet_test.go#L49-L50

https://github.com/fastly/cli/blob/6c1ec982475e698bd347c94622f857d9e4f44c90/pkg/commands/vcl/snippet/snippet_test.go#L206